support/ci/deploy_mac_launcher.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+# fail fast if we aren't on the desired branch or if this is a pull request
+if  [[ "${TRAVIS_BRANCH}" != "$(cat VERSION)" && ("${TRAVIS_PULL_REQUEST}" != "false" || "${TRAVIS_BRANCH}" != "master") ]]; then
+    echo "We only publish on successful builds of master."
+    exit 0
+fi
+
+BINTRAY_REPO=unstable
+if [ "$(cat VERSION)" == "$TRAVIS_TAG" ]; then
```

Comment:
Since we also use `$(cat VERSION)` above, it might be nice to pull that out as its own variable.